about.html

```diff
@@ -0,0 +1,130 @@
+
+        <!--Section flex container-->
+        <section class="flex-container">
+            <!--Hero-->
+
+            <div class="hero-container-about">
+            
+                <div class="hero-title-about">
+                    <h2>"Hello! Sharing Courses"</h2>
+                    <h1>Jose Delphin System Advisor Courses 2023</h1>
+                </div>
+                <div class="about-text">
+                    <p>The Jose Delphin System Advisor Courses 2023 brings together the community of  
+                        experts, to make available the best skills on HTML, CSS, and Javascript for all the world.
+                        This year, we hope to expand our invitation list including organizations and individuals.</p>
+                </div>
+                <div class="frame1">
+                    <p>Please contact us per Email for nay further questions about Jose Delphin System Advisor Courses 2023!</p>
+                    <a href="info@josedelphin.site">info@josedelphin.site</a>
+                </div>
+            </div>
+
+            
+            <div class="logo-info">
+                <h2>Jose Delphin System Advisor  <br>  Courses 2023 Logo</h2>
+                <hr>
+                <p>The logo of Jose Delphin System Advisor Courses 2023 was designed with the best practice of Photoshop.</p>
+                <img src="./assets/images/MiLogo.png" alt="logo">
+            </div>
+
+            <div class="previous-summit">
+                <h2>See the past Jose Delphin System Advisor</h2>
+                <p>Take a look at the past two Jose Delphin System Advisor Courses 2023 which took place in all the word in a remote way</p>
+                <div class="previous-img">
+                    <img src="./assets/images/img2011.png" alt="image 2011">
+                    <img src="./assets/images/img2013.png" alt="image 2013">
+                </div>
+            </div>
+
+            <!--Partners-->
+            <div class="partners-container">
+                <h2>Partner</h2>
+                <hr>
+                <div class="partners-logo">
+                    <img src="./assets/images/img-mozilla.png" alt="">
+                    <img src="./assets/images/img-google.png" alt="">
+                    <img src="./assets/images/img-naver.png" alt="">
+                    <img src="./assets/images/img-daumkakao.png" alt="">
+                    <img src="./assets/images/img-airbnb.png" alt="">
+                </div>
+            </div>
+        </section>
+        <footer class="footer-about">
+            <img src="./assets/images/cck-logo1.png" alt="cc logo">
+            <div class="footer-text">
+                <p>2015 Creative Commons Korea. Some Rights Reserved.</p>
+                <p class="korean-text">달리 정하지 않은 웹사이트의 모든 내용은 크리에이티브 커먼즈 저작자표시 2.0 대한민국 라이선스에 따라 이용할 수 있습니다.
+                    자세한 이용기능 내용과 허락조건은 http://creativecommons.org/licences/by/2.0/kr 를 보세요.</p>
+            </div>
+        </footer>
```

Comment:
- [ ] Great job adding the footer I just recommend removing the white space in the end since it doesn't exist in the original design and to improve user experience (UX) 💯✨🙏🏻✅

- I mean this white space

![image](https://user-images.githubusercontent.com/58553711/211971098-14a21129-6c12-4be8-968e-9fa850de1de8.png)

Reply:
Awesome. I am fixing it now. Thank you very much for your review